Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/model/DocumentMetadataResult.java

```diff
@@ -25,6 +25,7 @@
  */
 package gov.hhs.fha.nhinc.model;
 
+import java.util.Date;
```

Comment:
Is this so you can use it for the view?  Maybe we should have a separate view object from the database model object.